build.gradle.kts

```diff
@@ -64,6 +65,23 @@ dependencies {
     implementation("io.prometheus:prometheus-metrics-instrumentation-jvm:1.2.0")
     implementation("io.prometheus:prometheus-metrics-exporter-httpserver:1.0.0")
 
+    // NOTE:
+    //  We implicitly rely on Nabu's transitive dependency on Netty's public interfaces.
+    //  We could explicitly include Netty ourselves, but we prefer to make sure we use the same version as Nabu.
+
+    // NOTE:
+    //  For jitpack's syntax for GitHub version resolution, refer to: https://docs.jitpack.io/
+
+    // Nabu
+//    implementation("com.github.LimeChain:nabu:master-SNAPSHOT") // Uncomment for "most-recent on the master branch"
+    implementation("com.github.LimeChain:nabu:16c6586")
+
+    // Guava
```

Comment:
Those two are nonetheless included as transitive from Nabu, but we don't use them in any way related to that re-exporting occurring, so I've made them explicit. Just pointing that out in case anyone's opinion wants to differ on that one 😃